src/apache.c

```diff
@@ -586,14 +587,30 @@ static int apache_init (void) /* {{{ */
 {
 	/* Call this while collectd is still single-threaded to avoid
 	 * initialization issues in libgcrypt. */
+	if(cd_ssl_multithread_setup() == SSL_MT_OK){
```

Comment:
Hi @octo  .

Indeed this function have to be called from every plugin that uses libgcrypt , ie openssl . 

do you prefer to insert the INFO() and WARNING() messages on the    cd_ssl_multithread_setup() ?